views/pages/get-involved/volunteer-application.pug

```diff
@@ -39,7 +39,7 @@ block body
           label(for="coop-false") No
         
         label(for="interest" class="required") Please explain why you are interested in this volunteer role.
-        textarea(id="interest" class="text-input" name="interest" required maxlength="6000")
+        textarea(id="interest" class="text-input" name="interest" required maxlength="6000" pattern=".*[^ ].*" oninvalid="this.setCustomValidity('Please enter your answer')" onchange="try{setCustomValidity('')}catch(e){}" oninput="setCustomValidity(' ')")

```

Comment:
I thought I already did this 🧍 But it just adds custom error messages for invalid (empty) forms